pymilvus/client/abstract.py

```diff
@@ -234,10 +235,15 @@ def succ_index(self):
     def err_index(self):
         return self._err_index
 
+    @property
+    def cost(self):
```

Comment:
Please add comment to explain what does cost mean and what's the unit of cost. Is it time? is it memory related? or both? or other meanings?

Reply:
is vcu